backend/src/main/java/com/ody/meeting/repository/MeetingRepository.java

```diff
@@ -12,7 +13,15 @@ public interface MeetingRepository extends JpaRepository<Meeting, Long> {
             from Meeting meeting
             right join Mate mate on meeting.id = mate.meeting.id
             where mate.member.id = :memberId
+            and meeting.overdue = false
             """
     )
     List<Meeting> findAllByMemberId(Long memberId);
```

Comment:
[제안]

헷갈릴 수 있을 것 같은데, 메서드명을 바꿔주면 어떤가요? NotOverdue 키워드가 들어가면 좋을 것 같아요

```suggestion
            and meeting.overdue = false
            """
    )
    List<Meeting> findAllNotOverdueByMemberId(Long memberId);
```

Reply:
헷갈릴 수 있을 것 같다했는데
메서드를 만든 저부터 헷갈려서 Overdue가 아님에도 부정어를 붙이지 않았었네요 😅 반영완료 !